src/main/java/com/google/cloud/genomics/dataflow/utils/GenomicsOptions.java

```diff
@@ -33,28 +29,30 @@
  * annotated with a @JsonIgnore annotation.
  */
 public interface GenomicsOptions extends DataflowPipelineOptions {
+  
+  // Use the same appName for all pipelines in this collection so that the oauth flow only needs to happen once.
+  public static final String APP_NAME = "dataflow-java";
```

Comment:
I think the current usage is that new apps will use or extend GenomicsOptions. But that would mean they all get the same app name, doesn't it?

It seems more reasonable to have the app write its own name somewhere. Either in the options as today, or we could just make it an argument go getGenomicsAuth if you prefer.